lib/src/airship_channel.dart

```diff
@@ -34,15 +35,22 @@ class AirshipChannel {
   }
 
   /// Adds channel tags.
+  /// Deprecated. Use editTags() instead.
   Future<void> addTags(List<String> tags) async {
     return await _module.channel.invokeMethod('channel#addTags', tags);
   }
 
   /// Removes channel tags.
+  /// Deprecated. Use editTags() instead.
   Future<void> removeTags(List<String> tags) async {
     return await _module.channel.invokeMethod('channel#removeTags', tags);
   }
 
+  /// Creates a [TagEditor] to modify the device tags.
+  TagEditor editTags() {
```

Comment:
do we want to provide an API with closure that automatically call apply in the end? it was very handy to use in native sdks

Reply:
In my opinion, I think we want to keep the same structure than our other editors for this one, but I might be an idea for a next version